client/my-sites/domains/domain-management/dns/dns-add-new.jsx

```diff
@@ -266,7 +266,7 @@ class DnsAddNew extends React.Component {
 		);
 		const buttonLabel = recordToEdit
 			? translate( 'Update DNS record' )
-			: translate( 'Add new DNS record' );
+			: translate( 'Add DNS record' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Add New DNS Record' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/dns-pages-design-fixes)